src/java/org/apache/cassandra/io/sstable/Descriptor.java

```diff
@@ -315,6 +315,11 @@ public static Component validFilenameWithComponent(String name)
         }
     }
 
+    public static boolean validFilename(String name)
```

Comment:
add a test mentioning  so that this method is referenced (it is currently unused in C*)